pkg/region/region.go

```diff
@@ -18,7 +18,7 @@ type Name string
 
 // Region represents the members of the Region enumeration.
 type Region struct {
-	name                  string
+	Name                  string
```

Comment:
Do we need to expose this? (vs using `String()` in the test case above). IIRC this was not exported to prevent the default set from being modified at run-time (since they are structs and can't be const)

Reply:
👍